README.md

```diff
@@ -24,7 +24,7 @@ jobs:
     name: Set Asana task status
     steps:
       - name: Set Asana task status
-        uses: sam-peach/GitHubAsanaStatusAction@main
+        uses: onesignal/GitHubAsanaStatusAction@main
```

Comment:
💯

Reply:
Oops, just realized that `onesignal/GitHubAsanaStatusAction@main` isn't a valid repo.

Should be `onesignal/asanabot@main`